dl1_data_handler/image_mapper.py

```diff
@@ -2,9 +2,9 @@
 
 class ImageMapper():
 
-    def __init__(self, mapping_method, channels):#", settings=defaults, ..."):
-        self.mapping_method = mapping_method
-        self.channels = channels
+    def __init__(self, **kwargs):
+        self.mapping_method = kwargs['mapping_method']
```

Comment:
We could add a try to avoid the KeyError.
Can we also return the input vector of map_image if no mapping_method is provided ?

Reply:
I changed it to dict.get() with a default value of "vector".

Reply:
> Can we also return the input vector of map_image if no mapping_method is provided ?

I think this is a great suggestion. There could be a vector mapping method that does this and is the default.